package.json

```diff
@@ -106,7 +105,8 @@
       "index.ts",
       ".+\\/__tests__\\/.+\\.builder.(t|j)s",
       ".+\\/__tests__\\/.+\\.factory.(t|j)s",
-      ".e2e-spec.ts"
+      ".e2e-spec.ts",
+      "dist"
```

Comment:
Why should we include `dist` in the coverage?

Reply:
`dist ` is now included in `coveragePathIgnorePatterns`

Reply:
Apologies, I misread it. Is this change needed in this PR?

Reply:
Well, I added it because we had coverage errors. Not necessarily needed in this PR.

Reply:
Do you mean the errors we solved by tweaking the coverage settings of the repo? If so (and it doesn't reduce the coverage), I think it best to revert this (nit).

Reply:
Removed
https://github.com/safe-global/safe-client-gateway/pull/1988/commits/905b0aa9113a7c34e7325e3b6e91db7c500bf663